.github/CODEOWNERS

```diff
@@ -161,6 +161,7 @@
 /core/domain/value_generators_domain*.py @vojtechjelinek
 /core/storage/exploration/ @aks681
 /core/templates/dev/head/components/entity-creation-services/exploration-creation.service*.ts @aks681
+/core/templates/dev/head/components/entity-creation-services/exploration-creation.service-backend-api*.ts @aks681
```

Comment:
Please merge from upstream/develop, since the path values in this file have changed too. Also, we've recently changed the ownership of this section, so please change to @kevintab95

Reply:
Also the path should be `/core/templates/components/entity-creation-services/exploration-creation.service-backend-api*.ts`